src/style.css

```diff
@@ -0,0 +1,75 @@
+@import url("https://fonts.googleapis.com/css?family=Roboto:500&display=swap");
+
+* {
+  margin: 0;
+}
+
+.header {
+  box-sizing: border-box;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+
+}
+
+.nav-menu {
```

Comment:
the only styles you need here is
```
list-style: none;
display: flex;
```

Reply:
<ul> has it's own margins and paddings. I need to override them.